utils/nctl/sh/scenarios/itst13.sh

```diff
@@ -25,25 +25,69 @@ function main() {
     check_network_sync
     # 3. Stop the node
     do_stop_node '5'
-    # 4. wait auction_delay + 1
-    do_await_era_change '4'
-    # 5. Validate eviction occured
+    # 4. Wait until N+1
+    do_await_era_change '1'
+    # 4a. Wait 1 block to avoid missing latest switch block
+    await_n_blocks '1' 'true'
```

Comment:
So I put that in after the era change because there's a slight chance that the node we were hitting for the switch block didn't have it yet and it was displaying the switch block from the previous era.